src/tests/unit/tests/DetailsView/components/assessment-instance-details-column.test.tsx

```diff
@@ -23,12 +24,11 @@ describe('AssessmentInstanceDetailsColumn', () => {
         };
 
         const wrapper = Enzyme.shallow(<AssessmentInstanceDetailsColumn {...props} />);
-
         verifyBaseRender(wrapper, props);
 
-        const label = wrapper.find('.assessment-instance-label');
+        const label = wrapper.find(`.${styles.assessmentInstanceLabel}`);
         expect(label.getElement().props.children).toBe(props.labelText);
-        expect(label.getElement().props.className).toBe('assessment-instance-label');
+        expect(label.getElement().props.className).toBe(styles.assessmentInstanceLabel);
```

Comment:
isn't this kind of redundant? I mean, if this is not true, then the `find` in line 29 will not find anything in the first place, right?